app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
Skeleton, auto-generated helper files like `UsersHelper` that have no methods can safely be removed. 

Get in the habit of reviewing all files generated by the `rails g(enerate) ...` command and delete those that aren't needed (they can always be generated again later). Use `git status` to review all generated files before `git add`-ing. Open each of the generated files and try to determine if its needed.

By keeping the app free of unnecessary files like this it tends to make maintenance a little easier as the app contains less clutter.